ema_workbench/analysis/prim.py

```diff
@@ -1164,8 +1164,8 @@ def find_box(self):
             _logger.info(
                 (
                     "box does not meet threshold criteria, "
-                    "value is {}, returning dump box"
-                ).format(box.mean)
+                    f"value is {box.mean}, returning dump box"
+                )
```

Comment:
Does this also fit on a single line? If yes, same as above

Reply:
some lines could indeed be reduced, see the latest commit. I already ran black on the file as well.